cdc/cdc/capture/http_validator.go

```diff
@@ -87,12 +84,14 @@ func verifyCreateChangefeedConfig(ctx context.Context, changefeedConfig model.Ch
 	if changefeedConfig.SinkConfig != nil {
 		replicaConfig.Sink = changefeedConfig.SinkConfig
 	}
-	if len(changefeedConfig.IgnoreTxnStartTs) != 0 {
-		replicaConfig.Filter.IgnoreTxnStartTs = changefeedConfig.IgnoreTxnStartTs
-	}
-	if len(changefeedConfig.FilterRules) != 0 {
-		replicaConfig.Filter.Rules = changefeedConfig.FilterRules
-	}
+	/*
```

Comment:
ditto

Reply:
FilterRules are used to filter table.

Reply:
We can modify the filter table to filter keys (or keyspan) to filter keys that users do not need to synchronize.